src/pages/RepoPage/FailedTestsTab/FrameworkTabsCard/FrameworkTabs/FrameworkTabs.tsx

```diff
@@ -13,11 +13,27 @@ const Frameworks = {
 type FrameworkType = (typeof Frameworks)[keyof typeof Frameworks]
 
 const FrameworkCopy = {
+  [Frameworks.PYTEST]:
+    'pytest --cov --junitxml=junit.xml -o junit_family=legacy',
+  [Frameworks.VITEST]: 'vitest --reporter=junit',
+  [Frameworks.JEST]: (
+    <p data-testid="jest-framework-copy">
+      npm i --save-dev jest-junit <br />
+      JEST_JUNIT_CLASSNAME=
+      <span className="text-codecov-code">{'"{filepath}"'}</span> jest
+      --reporters=jest-junit
+    </p>
+  ),
+  [Frameworks.PHP_UNIT]: './vendor/bin/phpunit --log-junit junit.xml',
+} as const
+
+const FrameworkClipboard = {
   [Frameworks.PYTEST]:
     'pytest --cov --junitxml=junit.xml -o junit_family=legacy',
   [Frameworks.VITEST]: 'vitest --reporter=junit',
   [Frameworks.JEST]:
-    'npm i --save-dev jest-junit \njest --reporters=jest-junit',
+    'npm i --save-dev jest-junit \n JEST_JUNIT_CLASSNAME="{filepath}" jest --reporters=jest-junit',
+
   [Frameworks.PHP_UNIT]: './vendor/bin/phpunit --log-junit junit.xml',
 } as const
```

Comment:
could we use
```
const FrameworkClipboard = {
  ...FrameworkCopy,
  [Frameworks.JEST]: 'npm i --save-dev jest-junit \n JEST_JUNIT_CLASSNAME="{filepath}" jest --reporters=jest-junit',
}
? I think it would be cleaner and clearer

Reply:
I like that a lot!